velox/exec/HashTable.cpp

```diff
@@ -1126,17 +1136,19 @@ bool HashTable<ignoreNullKeys>::arrayPushRow(char* row, int32_t index) {
 }
 
 template <bool ignoreNullKeys>
-void HashTable<ignoreNullKeys>::pushNext(char* row, char* next) {
+void HashTable<ignoreNullKeys>::pushNext(
+    RowContainer* rows,
```

Comment:
Document that this row container should be the same as the one in hash table except for `parallelJoinBuild`